tasks/src/test/java/org/example/aProblemAboutEquation/AProblemAboutEquation3Test.java

```diff
@@ -0,0 +1,78 @@
+package org.example.aProblemAboutEquation;
+
+import lombok.extern.java.Log;
+import org.junit.Before;
+import org.junit.jupiter.api.*;
+
+import java.util.List;
+
+@Log
+class AProblemAboutEquation3Test {
+    private final AProblemAboutEquation3 aProblemAboutEquation3 = new AProblemAboutEquation3();
+
+
+    @Before
+    //method with @Before must be public
+    //Gdzie ta metoda się wykonuje, że nie widać komunikatów??
+    public void showMessage() {
+         log.info("this is my first test's class");
+        System.out.println("Gdzie ta metoda się wykonuje, że nie widać komunikatów??");
+        //Po co jest to flush???????
```

Comment:
https://stackoverflow.com/questions/19183955/why-we-use-system-out-flush

Reply:
Z tego, co zrozumiałem jest to zupełnie niepotrzebne przy System.out.flush().

Reply:
to prawda. Tam to z automatu flushuje java